test/svg-sprite.js

```diff
@@ -925,7 +936,7 @@ describe('svg-sprite', () => {
 
                         preview.should.be.ok;
 
-                        capturePhantom(preview, previewImage, error => {
+                        capturePuppeteer(preview, previewImage, error => {
```

Comment:
Have to mention that it's cool that you keep same interface here. 👍

Reply:
I tried to minimalise the modifications. I believe in small PRs so if someone wanna improve or rework things they can do it in other PRs.
I think would be good refactor the test because few times hard to figure it out which test failed which pixels are different.